src/NUnitFramework/nunitlite.tests/CommandLineTests.cs

```diff
@@ -172,7 +172,7 @@ public void CanRecognizeLowerCaseOptionValues(string propertyName, string option
 
             foreach (string canonicalValue in canonicalValues)
             {
-                string lowercaseValue = canonicalValue.ToLower(CultureInfo.InvariantCulture);
+                string lowercaseValue = canonicalValue.ToLower();
```

Comment:
The form of `ToLower()` that takes a `CultureInfo` is not available on .NET Standard so I dropped it rather than use another `#if`